src/github.com/matrix-org/dendrite/clientapi/routing/register.go

```diff
@@ -399,10 +400,27 @@ func Register(
 	// Retrieve or generate the sessionID
 	sessionID := r.Auth.Session
 	if sessionID == "" {
-		// Generate a new, random session ID
+		// Generate a new, random session ID`
 		sessionID = util.RandomString(sessionIDLength)
 	}
 
+	// Don't allow numeric usernames less than MAX_INT64.
```

Comment:
greater than? :P

Reply:
No ?
We are using numbers less than int_max 64, so only disallow numbers less than the same.

Reply:
Ah, for users trying to register a username. Thought this was referencing generated usernames.